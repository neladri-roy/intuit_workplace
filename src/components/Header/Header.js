import React from 'react'
import { Wrapper, HeaderContainer, Title } from './Header.style'
import { Link } from 'react-router-dom';
const Header = () => {
    return (
      <Wrapper>
      
          <HeaderContainer data-testid="header">
          <Link to="/">
          <Title>Work@Tech</Title>
          </Link>
          </HeaderContainer>  
          
      </Wrapper>
    );
}

export default Header
