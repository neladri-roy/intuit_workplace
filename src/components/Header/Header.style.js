import styled from "styled-components";


export const Wrapper = styled.div`
    width: 100%;
    position: fixed;
    top: 0;
    left: 0;
    transition: 0.5s;
    backdrop-filter: blur(3px);
`;


export const HeaderContainer = styled.div`
height: 15vh;
position:sticky;
width: 100%;
`;


export const Title = styled.h1`
    color: ${(props) => props.theme.TitleColor};
    font-weight: 600;
    font-size: 3.6rem;
    padding: 2rem 2rem;
`

