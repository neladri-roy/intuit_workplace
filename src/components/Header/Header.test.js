
import Header from "./Header";
import { screen } from '@testing-library/react';
import { shallow } from "enzyme";
import React from 'react';

test('renders title ', ()=>{
   const component = shallow(<Header/>);
   expect(component).toMatchSnapshot();
})