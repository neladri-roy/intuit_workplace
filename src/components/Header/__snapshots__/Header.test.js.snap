// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders title  1`] = `ShallowWrapper {}`;
